src/main/scala/dpla/ingestion3/ReporterMain.scala

```diff
@@ -26,10 +26,11 @@ import util.{Try, Failure}
   *
   * @see ReporterTest
   *
-  * @param token            The report name token
+  *
   * @param inputURI         Input URI or file path
   * @param outputURI        Output URI or file path
   * @param sparkMasterName  Spark master name, e.g. "local[1]"
+  * @param token            The report name token
```

Comment:
This reordering was just to bring it in line with how it expects the parameters to be ordered.